src/middlewares/pre/index.ts

```diff
@@ -0,0 +1,7 @@
+import CrossOriginHeadersMiddleware from '../pre/cors/index';
+import RequestDataJsonParserMiddleware from './bodyParser';
+
+export default [
+  CrossOriginHeadersMiddleware(`${process.env.LOCALHOST}`),
```

Comment:
env z dupy nazwany, moze lepiej clientAdress?

Reply:
Tez tak uwazam +++ skoro w configu zaciagasz wszystkie envy, to fajnie byloby jakbys tam zaciagnal tez tego enva a tutaj zaciagnal z configu.